### DISCUSSION TITLE
Craft CMS 5.0 alpha 7 icon field - for front-end and GraphQL?

### DISCUSSION THREAD

**Discussion by davidwebca** [craftcms/cms#14192] [Q&A] [answered]
Hi @brandonkelly,

I know it's been released only yesterday, but we're currently working on a website that will be released down the road after Craft 5's release so we already started working with the alpha to bite the bullet and get ahead.

Question: Will this new field allow us to customize the fonts themselves and also return a value through GraphQL?

I've just tested the field and looked quickly through the source code and it seems to be pretty "baked" with the pre-packaged icon set and the value of the field returned is only the name of the file without extension, which is great in itself and would be easy to use through twig. 

We're currently using verbb's icon-picker field but we're always excited to cut down plugins when something native gets added.

Thanks a lot and continue the great work! 💪

**Comment by brandonkelly** [comment, marked as answer]
The field values will be set to [Font Awesome](https://fontawesome.com/) icon names, so it will only really be useful on the front-end if you’re using that. Not actually possible to get the icon SVG contents directly via the field value.

**Reply by davidwebca** [reply]
Gotcha. Thanks for the fast reply! 👍
